testpmd-container-app/cnfapp/Dockerfile

```diff
@@ -1,20 +1,32 @@
 FROM registry.redhat.io/openshift4/dpdk-base-rhel8:v4.6.3-8
 
-MAINTAINER skramaja@redhat.com
+MAINTAINER telcoci@redhat.com
 
 LABEL name="NFV Example CNF Application" \
-      maintainer="skramaja@redhat.com" \
+      maintainer="telcoci@redhat.com" \
       vendor="fredco" \
-      version="v0.2.1" \
-      release="v0.2.1" \
+      version="v0.2.5" \
+      release="v0.2.5" \
       summary="An example CNF for platform valiation" \
       description="An example CNF for platform valiation"
 
 COPY licenses /licenses
 
 USER root
-RUN yum -y update-minimal --disablerepo "*" --enablerepo rhel-8-for-x86_64-baseos-rpms \
-        --enablerepo rhel-8-for-x86_64-appstream-rpms --setopt=tsflags=nodocs \
-        --security --sec-severity=Important --sec-severity=Critical
 
-USER default
+# Install Go binary
+ARG GO_VERSION=go1.21.6
+ADD https://go.dev/dl/$GO_VERSION.linux-amd64.tar.gz /usr/local
+RUN tar -C /usr/local -xzf /usr/local/$GO_VERSION.linux-amd64.tar.gz
+RUN rm /usr/local/$GO_VERSION.linux-amd64.tar.gz
+ENV PATH="$PATH:/usr/local/go/bin"
```

Comment:
There's no need to install go in here, use two container instead, one for build and one to ship, only copy the binary in the last one (to ship), like this:

```Dockerfile
FROM docker.io/library/golang:1.21 as builder

WORKDIR /scripts
COPY scripts/webserver.go .
RUN go mod init webserver.go
RUN GOOS=linux CGO_ENABLED=0 go build -a -o webserver .

# This image needs confirmation :warning:
FROM registry.redhat.io/openshift4/dpdk-base-rhel8:v4.6.3-8

-- snip --

COPY chmod u+x scripts/entrypoint.sh  /usr/local/bin/
COPY chmod u+x --from=builder /scripts/webserver /usr/local/bin/

ENTRYPOINT ["/usr/local/bin/entrypoint.sh"]
```

This way you don't need to install anything as I mentioned previously, just copy the binary.

Reply:
I think I have done this correctly, please confirn